notebooks/CubevizExample.ipynb

```diff
@@ -27,13 +63,139 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from astropy.utils.data import download_file\n",
-    "\n",
-    "# This file is originally from https://data.sdss.org/sas/dr14/manga/spectro/redux/v2_1_2/7495/stack/manga-7495-12704-LOGCUBE.fits.gz\n",
-    "# but has been modified to correct some inconsistencies in the way units are parsed\n",
     "fn = download_file('https://stsci.box.com/shared/static/28a88k1qfipo4yxc4p4d40v4axtlal8y.fits', cache=True)\n",
-    "viz.load_data(fn)"
+    "\n",
+    "with warnings.catch_warnings():\n",
+    "    warnings.simplefilter('ignore')\n",
+    "    cubeviz.load_data(fn)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The upper left viewer containing the FLUX cube is the default viewer that is accessible as follows."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "viewer = cubeviz.default_viewer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note also that in the above example there are mouse-over coordinates visible by default."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It possible to make selections/regions in images and export these to astropy regions. Click on an image viewer toolbar then click on the circular selection tool, and drag and click to select an interesting region on the sky. We can then export this region with:"
```

Comment:
```suggestion
    "It possible to make selections/regions in images and export these to `astropy` regions. Click on an image viewer toolbar then click on the circular selection tool, and drag and click to select an interesting region on the sky. We can then export this region with:"
```

Reply:
I would rather do Astropy because `regions` is not in `astropy` core library, but rather part of the Astropy project. I will capitalize.